lib/inc/facter/execution/execution.hpp

```diff
@@ -11,10 +11,16 @@
 #include <utility>
 #include <stdexcept>
 #include <functional>
-#include "../util/option_set.hpp"
-#include "../util/environment.hpp"
 #include "../export.h"
 
+#include <leatherman/util/environment.hpp>
+#include <leatherman/util/option_set.hpp>
+#include <leatherman/execution/execution.hpp>
+#include <leatherman/execution/execution.hpp>
+
+namespace lth_util = leatherman::util;
+namespace lth_exec = leatherman::execution;
+
 namespace facter { namespace execution {
 
     /**
```

Comment:
What is the best way to handle all of these exceptions and whatnot? They currently appear both here and in Leatherman.